test/acceptance/behave/environment.py

```diff
@@ -79,6 +82,7 @@ def after_feature(context, feature):
     :param feature: feature properties
     """
     context.execute_steps(u'Given stop service')
+    print colored('    And stop context Broker', 'green')
```

Comment:
this print is used to show some steps that are execute manually and it is not a normal `print` use `termcolor` library.